crates/trippy-tui/locales/app.yml

```diff
@@ -599,6 +658,7 @@ help_show_bindings:
   sv: "Tryck på [%{key}] för att visa alla kortkommando"
   ru: "Нажмите [%{key}], чтобы показать все привязки"
   es: "Presiona [%{key}] para mostrar todos los atajos"
+  de: "Drücken Sie [%{key}], um alle Bindungen anzuzeigen"
```

Comment:
I would rather translate this with "Tastaturkürzel" (keyboard shortcuts) or "Tastaturbelegungen" (Keyboard bindings) to make clear that its about key shortcuts.
"Drücken Sie [%{key}], um alle Tastaturkürzel anzuzeigen"

Reply:
I saw somewhere else already "Tastenbelegungen", that would be a good fit.

Reply:
So i'm clear then, "Drücken Sie [%{key}], um alle Tastenbelegungen anzuzeigen"?  Should "Tastenbelegungen" be capitalized?

Reply:
@peshay can you confirm on this one?

Reply:
Yes, that is correct. Tastenbelegungen capitalized